index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     }
 
     this.versionChecker = new VersionChecker(this);
-    this.versionChecker.for('ember-cli', 'npm').assertAbove('0.2.0');
+    this.versionChecker.forEmber().assertAbove('0.2.0');
```

Comment:
Oh, I see. But I have a build error with liquid-fire similar to this https://github.com/yapplabs/ember-modal-dialog/pull/166 after updating to ember-cli 2.12. I don't know what else could be the problem.